src/init.cpp

```diff
@@ -458,6 +458,16 @@ std::string HelpMessage(HelpMessageMode mode)
     const auto testnetBaseParams = CreateBaseChainParams(CBaseChainParams::TESTNET);
     const auto defaultChainParams = CreateChainParams(CBaseChainParams::MAIN);
     const auto testnetChainParams = CreateChainParams(CBaseChainParams::TESTNET);
+
+    Consensus::Params devnetConsensus;
+    Consensus::LLMQParams devnetLLMQ;
+    if (gArgs.IsArgSet("-devnet")) {
+        devnetConsensus = CreateChainParams(CBaseChainParams::DEVNET)->GetConsensus();
+        devnetLLMQ = devnetConsensus.llmqs.at(Consensus::LLMQ_DEVNET);
+    }
+
+    const auto& regtestLLMQ = CreateChainParams(CBaseChainParams::REGTEST)->GetConsensus().llmqs.at(Consensus::LLMQ_TEST);
```

Comment:
Maybe I'm missing something, but I don't see how this is relevant

Reply:
It's used for `-llmqtestparams` help sptring